_source/handbook/00_the_origin_of_stimulus.md

```diff
@@ -1,5 +1,6 @@
 ---
-permalink: /handbook/origin
+title: The Origin of Stimulus
+permalink: /handbook/origin.html
```

Comment:
Is the addition of `.html` required? Doesn't seem to be reflected in the actual permalink.

Reply:
This is largely due to [how 11ty creates permalinks](https://www.11ty.dev/docs/permalinks/). It won't actually make a page if you don't include the extension. I could change the URL structure to be more inline with 11ty defaults, but I thought it'd be better to preserve URLs as they were.